cmake/cc_rules.cmake

```diff
@@ -225,33 +233,59 @@ function(firebase_ios_maybe_remove_objc_sources output_var)
   set(${output_var} ${sources} PARENT_SCOPE)
 endfunction()
 
-# firebase_ios_add_objc_flags(target sources...)
+# firebase_ios_add_compile_options(target [DISABLE_STRICT_WARNINGS] sources...)
+#
+# Adds FIREBASE_IOS_CXX_FLAGS or FIREBASE_IOS_CXX_FLAGS_STRICT to the compile
+# options of the given target depending on whether or not
+# DISABLE_STRICT_WARNINGS was passed.
 #
-# Adds FIREBASE_IOS_OBJC_FLAGS to the compile options of the given target if
-# any of the sources have filenames that indicate they are Objective-C.
-function(firebase_ios_add_objc_flags target)
-  set(_has_objc OFF)
+# If any of the sources have filenames that indicate they are Objective-C adds
+# Either FIREBASE_IOS_OBJC_FLAGS or FIREBASE_IOS_OBJC_FLAGS_STRICT depending on
+# whether or not DISABLE_STRICT_WARNINGS was passed.
+function(firebase_ios_add_compile_options target)
+  set(flag DISABLE_STRICT_WARNINGS)
+  cmake_parse_arguments(aco "${flag}" "" "${}" ${ARGN})
```

Comment:
Is there a reason for the empty brackets?

Reply:
No. This is a mistake (though I'm surprised it works). Fixed.